spec/takeaway_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new(menu) }
+  let(:menu) { double :menu }
+
+  describe '#show_menu' do
+
+    it 'returns an empty hash' do
+      allow(menu).to receive(:show).and_return({})
+      expect(takeaway.show_menu).to eq ({})
+    end
+  end
+
+  describe '#select' do
+    it 'selects a few dishes from the menu' do
+      allow(menu).to receive(:show).and_return({ ribs: 3,
+      beef: 4,
+      rolls: 3,
+      chips: 2,
+      pies: 5})
+      expect(takeaway.select({beef: 2, rolls: 3}, 20)).to eq ({beef: 2, rolls: 3})
```

Comment:
Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)<br>Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/RedundantParentheses: Don't use parentheses around a literal.